include/seqan3/range/view/translation.hpp

```diff
@@ -45,292 +45,684 @@
+ * | seqan3::const_iterable_concept      |                                       | *preserved*                                        |
+ * |                                     |                                       |                                                    |
+ * | seqan3::reference_t                 | seqan3::nucleotide_concept            | seqan3::aa27                                       |
+ *
+ * * `urng_t` is the type of the range modified by this view (input).
  * * `rrng_type` is the type of the range returned by this view.
  * * for more details, see \ref view.
  *
- * \par Example
+ * ### Example
  *
  * Operating on a range of seqan3::dna5:
  * ```cpp
  * dna5_vector vec{"ACGTACGTACGTA"_dna5};
  *
- * // single frame translation
- * auto v1 = vec | view::translate_single;                     // == [T,Y,V,R]
+ * // Default (first forward frame)
+ * auto v1 = vec | view::translate_single;                                                           // == [T,Y,V,R]
+ *
+ * // Default (first forward frame)
+ * auto v2 = vec | view::translate_single();                                                         // == [T,Y,V,R]
+ *
+ * // First forward frame
+ * auto v3 = vec | view::translate_single(translation_frames::FWD_FRAME_0);                          // == [T,Y,V,R]
+ *
+ * // First reverse frame
+ * auto v4 = vec | view::translate_single(translation_frames::REV_FRAME_0);                          // == [Y,V,R,T]
+ *
+ * // Second forward frame
+ * auto v5 = vec | view::translate_single(translation_frames::FWD_FRAME_1);                          // == [R,T,Y,V]
+ *
+ * // Second reverse frame
+ * auto v6 = vec | view::translate_single(translation_frames::REV_FRAME_1);                          // == [T,Y,V,R]
+ *
+ * // Third forward frame
+ * auto v7 = vec | view::translate_single(translation_frames::FWD_FRAME_2);                            // == [V,R,T]
+ *
+ * // Third reverse frame
+ * auto v8 = vec | view::translate_single(translation_frames::REV_FRAME_w);                            // == [R,T,Y]
  *
  * // function syntax
- * auto v2 = view::translate_single(vec);                      // == [T,Y,V,R]
+ * auto v9 = view::translate_single(vec, translation_frames::FWD_FRAME_0);                           // == [T,Y,V,R]
  *
  * // combinability
- * auto v3 = vec | view::complement | view::translate_single;  // == [C,M,H,A]
+ * auto v10 = vec | view::complement | view::translate_single(translation_frames::REV_FRAME_0);      // == [M,H,A,C]
+ *
+ * // combinability with default parameter
+ * auto v11 = vec | view::complement | view::translate_single;                                       // == [C,M,H,A]
+ *
+ * // combinability with default parameter
+ * auto v12 = vec | view::complement | view::translate_single();                                     // == [C,M,H,A]
+ *
  * ```
  */
-
 seqan3::detail::translate_single_fn const translate_single;
```

Comment:
as per #268, please add "inline constexpr" for these, e.g. `inline constexpr seqan3::detail::translate_single_fn translate_single;`